app/[event]/event.tsx

```diff
@@ -595,10 +595,11 @@ const styles = {
         marginBottom: '1rem',
     },
     coverPhoto: {
-        width: '100%',
-        height: 'auto',
+        maxWidth: '100%',
+        maxHeight: '600px',
         margin: '1rem 0',
         borderRadius: '4px',
+        resizeMode: 'center',
```

Comment:
does resizeMode do anything?

preview lgtm but i don't see resizeMode in the final css

Reply:
oh good catch, it should determine the stretching of the image if its smaller than the container. center will make sure the image never gets stretched. It looks like the style was deprecated in lieu of a HTML attr though

Reply:
https://www.w3schools.com/jsref/prop_style_objectfit.asp

Used object fit instead